cluster.go

```diff
@@ -15,6 +16,15 @@ func (c *ClientIMPL) GetCluster(ctx context.Context) (resp Cluster, err error) {
 	var systemList []Cluster
 	cluster := Cluster{}
 	qp := c.APIClient().QueryParamsWithFields(&cluster)
+
+	majorVersion, err := c.GetSoftwareMajorVersion(ctx)
+	if err != nil {
+		log.Printf("ERROR: Couldn't find the major array version")
```

Comment:
Please change it info, error or debug statements, Printf logs may not appear when we set `X_CSI_DEBUG` to true

Reply:
Updated with log.Errorf statements